editor.planx.uk/src/pages/Team/components/FlowCard.tsx

```diff
@@ -145,6 +145,7 @@ const FlowCard: React.FC<FlowCardProps> = ({
 
   const isSubmissionService = flow.publishedFlows?.[0]?.hasSendComponent;
   const isTemplateService = Boolean(flow.templatedFrom);
+  const isSourceTemplateService = Boolean(flow.isTemplate);
```

Comment:
nit: `flow.isTemplate` is naturally type boolean, do we actually need to wrap it in `Boolean()` again here?

Reply:
Good catch (again! 😅), looks like a heavy-handed `Cmd + Shift + D` on my end here!